### DISCUSSION TITLE
Hide times in overview

### DISCUSSION THREAD

**Discussion by Disane87** [totaldebug/atomic-calendar-revive#1493] [Support]
I have a calander via iCal which was provides by my local waste disposal company. These appointsments aren't all day events, they are normal appointments with a start and end date. Ist it possible to hide the times in the second row to get some more space? This would be pretty useful to show this calendar on a screen.

![image](https://github.com/totaldebug/atomic-calendar-revive/assets/956158/dee3fcec-1c8b-4498-be72-e259c299b510)

**Comment by marksie1988** [comment]
no its not currently possible to hide the times for events. 

you could possibly add some CSS to hide the div though.
